src/examples/forms/range/RangeDefault.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Field, Label, Range } from '@zendeskgarden/react-forms';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => {
+  const [rangeValue, setRangeValue] = useState(150);
+
+  return (
+    <Row>
+      <Col>
+        <Field>
+          <Label>Flowers</Label>
+          <Range
+            min={0}
+            aria-valuemin={0}
+            max={300}
+            aria-valuemax={300}
+            value={rangeValue}
+            aria-valuenow={rangeValue}
+            step={10}
+            onChange={event => setRangeValue(parseInt(event.target.value, 10))}
+          />
+        </Field>
+      </Col>
+    </Row>
+  );
+};
```

Comment:
It would be helpful to show a dead-simple, uncontrolled example here:

```
const Example = () => (
  <Row>
    <Col>
      <Field>
        <Label>Flowers</Label>
        <Range step={10} />
      </Field>
    </Col>
  </Row>
);
```

Reply:
Ah yes, that's because the `Range` component is backed by a native slider, it has a `type='range'`.